uber/models.py

```diff
@@ -1654,6 +1654,14 @@ class MPointsForCash(MagModel):
     when        = Column(UTCDateTime, default=lambda: datetime.now(UTC))
 
 
+class PromoCode(MagModel):
+    price           = Column(Integer, default=0)
+    uses            = Column(Integer, default=1)
+    code            = Column(UnicodeText)
+    users           = Column(UnicodeText, default="")
```

Comment:
Gotcha.  That also works, but is only better if we expect people to be able to apply multiple different promo codes.  If we want to limit it to one promo code per registration, then a foreign key on the ``Attendee`` model would be better.  Otherwise a ``PromoCodeUsage`` model is better.